subsys/net/lib/lwm2m/lwm2m_obj_server.c

```diff
@@ -33,8 +39,24 @@ LOG_MODULE_REGISTER(LOG_MODULE_NAME);
 #define SERVER_STORE_NOTIFY_ID		6
 #define SERVER_TRANSPORT_BINDING_ID	7
 #define SERVER_REG_UPDATE_TRIGGER_ID	8
+#if defined(CONFIG_LWM2M_SERVER_OBJECT_VERSION_1_1)
+#define SERVER_BOOTSTRAP_UPDATE_TRIGGER_ID	9
```

Comment:
Would be nice to keep the numbers aligned.

Reply:
OK I will do that.